### ISSUE TITLE
state_sync_* nightly tests are failing

### ISSUE THREAD

**Issue by mfornet** [nearprotocol/nearcore#2734]
Some nightly tests related to state_sync are failing due to RPC becoming not responsive, possibly due to the load of state sync.

Failing tests:
- state_sync_routed [nightly](http://nightly.neartest.com/test/pytest/pytest_sanity_state_sync_routed_py_manytx_100)
- state_sync_late [nightly](http://nightly.neartest.com/test/pytest/pytest_sanity_state_sync_late_py_notx)
- state_sync_massive #2733

First two tests started failing consistently after #2672

**Comment by SkidanovAlex** [comment]
Duplicate of #2722
